csrankings-b.csv

```diff
@@ -803,10 +803,10 @@ Brygg Ullmer,Louisiana State University,https://www.lsu.edu/eng/cse/people/facul
 Bu-Sung Lee,Nanyang Technological University,http://research.ntu.edu.sg/expertise/academicprofile/Pages/StaffProfile.aspx?ST_EMAILID=ebslee,TyytTfQAAAAJ
 Bud Mishra,New York University,https://cs.nyu.edu/mishra,kXVBr20AAAAJ
 Budi Arief,University of Kent,https://www.cs.kent.ac.uk/people/staff/ba284,bSZk250AAAAJ
-Bugra Gedik,Bilkent University,http://www.cs.bilkent.edu.tr/~bgedik/homepage/doku.php,ddLcpMQAAAAJ

```

Comment:
Please add this and any similar entries to `old/industry.csv` with the affiliation at the end. Thanks.

Reply:
updated